src/popup/pages/Home.tsx

```diff
@@ -75,10 +82,14 @@ export const Component = () => {
             step={Number(minRateOfPay)}
             value={Number(rateOfPay)}
           />
-          <div className="flex w-full items-center justify-between px-2">
+          <div className="flex w-full items-center justify-between px-2 tabular-nums">
             <span className="text-sm">
               {rate} {getCurrencySymbol(walletAddress.assetCode)} per hour test
```

Comment:
Is that text stray: "test"?
```suggestion
              {rate} {getCurrencySymbol(walletAddress.assetCode)} per hour
```